src/hb-buffer-private.hh

```diff
@@ -344,7 +344,7 @@ struct hb_buffer_t {
   inline void
   unsafe_to_break_all (void)
   {
-    for (unsigned int i = 0; i < len; i++)
```

Comment:
This is not correct. It must scan over the buffer first and find minimum cluster number... scratch that. Remove the loop and just call `unsafe_to_break_impl (0, buffer->len)`.

Reply:
I’ve made this change.